conf/broker.conf

```diff
@@ -112,6 +112,17 @@ maxUnackedMessagesPerConsumer=50000
 # check and dispatcher can dispatch messages without any restriction
 maxUnackedMessagesPerSubscription=200000
 
+# Max number of unacknowledged messages allowed per broker. Once this limit reaches, broker will stop dispatching
+# messages to all shared subscription which has higher number of unack messages until subscriptions start
+# acknowledging messages back and unack count reaches to limit/2. Using a value of 0, is disabling
+# unackedMessage-limit check and broker doesn't block dispatchers
+maxUnackedMessagesPerBroker=30000000
```

Comment:
Should we merge this with default to 0 initially? And maybe enable by default in next release?